### ISSUE TITLE
ie11 not work

### ISSUE THREAD

**Issue by A50** [AlexRex/angular-event-service#2]
When npm start then ie11 error -> SCRIPT5007
row 54: 
	modules[moduleId].call(module.exports, module, module.exports, __webpack_require__);
In chrome, firefox is work and not error.

**Comment by AlexRex** [comment]
I don't think this is related to the library. Under the hood it's using plain rx/js observables.

**Comment by jfftck** [comment]
You need to `import { Observable } from 'rxjs/Observable'`, `import { Subject } from 'rxjs/Subject'`, and `import 'rxjs/add/observable/from'` instead of `import * as Rx from 'rxsj/Rx'` to have it work in IE. This is a known workaround.

I also had to point to the source index.ts in the package.json file to get IE to work, so it is your code and how you are compiling.

Please read how to build libraries for Angular 2+, since I do not have the time to fix the tsconfig.josn with the correct compiling options.
